drivers/dai/intel/dmic/dmic_nhlt.h

```diff
@@ -0,0 +1,119 @@
+/* SPDX-License-Identifier: Apache-2.0 */
```

Comment:
I know this is nit pick, but I've needed to explain the confusion many times to many folks: these headers are intel specific headers, in public nhlt spec they are not present, but only described as bespoke "vendor blobs" -> perhaps filename like dmic_intel.h would be more suitable etc.